src/test/resources/application.properties

```diff
@@ -18,4 +18,7 @@ scheduled.fixed.delay=600
 #spring.rabbitmq.username=guest
 #spring.rabbitmq.password=guest
 
-spring.datasource.initialization-mode=never
\ No newline at end of file
+spring.datasource.initialization-mode=never
+
+rest-template.connection.timeout=30
```

Comment:
Curious. Were default values not enough?

Reply:
without that the app for integration tests is not starting:
![image](https://user-images.githubusercontent.com/47707682/82324212-8f56ba00-99d9-11ea-9cbe-0999ef73be77.png)